tests/safeds/data/tabular/transformation/test_sequential_table_transformer.py

```diff
@@ -0,0 +1,215 @@
+        imputer = SimpleImputer(SimpleImputer.Strategy.constant(0))
+        transformer_list = [one_hot, imputer]
+        test_table = Table(
+            {
+                "col1": [1, 2, None],
+                "col2": ["a", "b", "a"],
+            },
+        )
+        sequential_table_transformer = SequentialTableTransformer(transformers=transformer_list)
+        old_hash = hash(sequential_table_transformer)
+        sequential_table_transformer.fit(test_table)
+        assert old_hash == hash(sequential_table_transformer)
+
+
+class TestTransform:
+    def test_should_raise_if_not_fitted(self) -> None:
+        one_hot = OneHotEncoder()
+        imputer = SimpleImputer(SimpleImputer.Strategy.constant(0))
+        transformers = [one_hot, imputer]
+        test_table = Table(
+            {
+                "col1": [1, 2, None],
+                "col2": ["a", "b", "a"],
+            },
+        )
+        sequential_table_transformer = SequentialTableTransformer(transformers)
+        with pytest.raises(TransformerNotFittedError, match=r"The transformer has not been fitted yet."):
+            sequential_table_transformer.transform(test_table)
+
+    @pytest.mark.parametrize(
+        "transformer",
+        [
+            OneHotEncoder(),
+            SimpleImputer(SimpleImputer.Strategy.constant(0)),
+            LabelEncoder(),
+            SimpleImputer(SimpleImputer.Strategy.mean()),
+        ],
+        ids=["OneHotEncoder", "Imputer with Constant", "LabelEncoder", "Mean Imputer"],
+    )
+    def test_should_do_same_as_transformer_with_single_transformer(self, transformer: TableTransformer) -> None:
+        sequential_transformer = SequentialTableTransformer([transformer])
+        test_table = Table(
+            {
+                "col1": [1, 2, None],
+                "col2": ["a", "b", "a"],
+            },
+        )
+        sequential_transformer = sequential_transformer.fit(test_table)
+        transformer = transformer.fit(test_table)
+        test_table_normal = transformer.transform(test_table)
+        test_table_sequential = sequential_transformer.transform(test_table)
+        assert_tables_equal(test_table_normal, test_table_sequential)
+
+    def test_transforms_correctly_with_multiple_transformers(self) -> None:
+        one_hot = OneHotEncoder()
+        imputer = SimpleImputer(SimpleImputer.Strategy.constant(0))
+        transformers = [one_hot, imputer]
+        test_table = Table(
+            {
+                "col1": [1, 2, None],
+                "col2": ["a", "b", "a"],
+            },
+        )
+        sequential_table_transformer = SequentialTableTransformer(transformers)
+        fitted_sequential_table_transformer = sequential_table_transformer.fit(test_table)
+        transfromed_table_sequential = fitted_sequential_table_transformer.transform(test_table)
+
+        one_hot = one_hot.fit(test_table)
+        transormed_table_individual = one_hot.transform(test_table)
```

Comment:
```suggestion
        transformed_table_individual = one_hot.transform(test_table)
```

Use rename refactoring.

Reply:
fixed typo